### DISCUSSION TITLE
Drop evince in favour of a native PDF app

### DISCUSSION THREAD

**Discussion by hanaral** [elementary/triage#118] [Ideas]
I know that it 'technically works' but it has at best had visual inconsistencies + bugs and is an overall pretty poor experience in my own experience. Aesop by Lains worked pretty well for me before, and it might serve well as a base to make a builtin first-party app that actually looks like and follows elementary guidelines.

**Comment by cassidyjames** [comment]
Evince works, is used on desktops like Red Hat and Ubuntu, and has GNOME developers developing and maintaining it. And with the new stylesheet, GNOME apps look more native than ever before. Unless you're volunteering to develop and maintain a new PDF viewer app, I don't think this is a good idea.

**Reply by hanaral** [reply]
I wasn't trying to imply that this needs to be a target for 6, but whenever someone is interested it might be good to post relevant stuff here for them to use.
Here are some of the visual ickynesses of Evince:



. | screenshot
------------ | -------------
example 1 | ![2021-04-19 20 15 03 のスクリーンショット](https://user-images.githubusercontent.com/58219504/115291566-d2bb2a80-a14c-11eb-9ced-069907320d6e.png)
example 2 | ![2021-04-19 20 15 25 のスクリーンショット](https://user-images.githubusercontent.com/58219504/115291568-d353c100-a14c-11eb-83ad-91d01e1a8403.png)
example 2 | ![2021-04-19 20 15 50 のスクリーンショット](https://user-images.githubusercontent.com/58219504/115291569-d353c100-a14c-11eb-9485-3a53204bddbc.png)

**Reply by cassidyjames** [reply]
Please file these as issues against the stylesheet. It's not strictly Evince's fault that we don't yet support widgets used in these ways.

**Comment by edmundlaugasson** [comment]
Would propose [Atril](https://github.com/mate-desktop/atril) as default PDF-reader. This is also using GTK library. Atril has simple UI but yet many advanced features. Also there is possible to configure UI so, that more buttons are visible.

**Comment by danirabbit** [comment]
I think it’s likely we’ll switch to [Papers](https://apps.gnome.org/Papers/) with a patch to use Granite’s stylesheet like we do with other GNOME apps we ship. It’s a much more modern design that solves a lot of the ickier UI/UX issues that were present in Evince
